release_notes.md

```diff
@@ -3,6 +3,8 @@
 <!-- Please add your release notes in the following format:
 - My change description (#PR/#issue)
 -->
+### Microsoft.Azure.Functions.Worker.Extensions.SignalRService <version>
```

Comment:
Extensions are released independently and have their own release notes. Please create a `extensions/Worker.Extensions.SignalRService/release_notes.md` and move these changes there.

See an example here: https://github.com/Azure/azure-functions-dotnet-worker/blob/main/extensions/Worker.Extensions.CosmosDB/release_notes.md

Reply:
SignalR should already have its own release note file

Reply:
Updated